backend/src/nodes/utils/utils.py

```diff
@@ -316,3 +317,21 @@ def alphanumeric_sort(value: str) -> List[Union[str, int]]:
     parts = NUMBERS.split(lcase_value)
     parts[1::2] = map(int, parts[1::2])
     return parts  # type: ignore
+
+
+def walk_error_handler(exception_instance):
+    logger.warning(
+        f"Exception occurred during walk: {exception_instance} Continuing..."
+    )
+
+
+def walk_sorted(path: str, ext_filter: Union[List[str], None] = None) -> List[str]:
```

Comment:
`walk_sorted` doesn't really tell you what the method does. Maybe something like `list_all_files_sorted` would be better.

Also, `path` is a very generic name as well. Do we expect this to be a directory or can it be a file as well?